sdk/storage/AzureStorageBlob/Source/Transport/URLSessionTransferManager.swift

```diff
@@ -27,7 +27,9 @@
 import AzureCore
 import CoreData
 
-final class URLSessionTransferManager: NSObject, TransferManager, URLSessionTaskDelegate {
+// swiftlint:disable type_body_length file_length
```

Comment:
Probably not :p But I wanted to keep refactoring to satisfy lint for a separate PR.

Reply:
Maybe it would be best to just not add the disable. Even if we address it sometime down the road, if we disable it now, we'll probably never come back to it.

Reply:
Resolved.